core/src/streams/clients.rs

```diff
@@ -253,11 +253,23 @@ impl StreamsClients {
                     chunk,
                 );
 
+                let alias_name = config
```

Comment:
may as well support this on all the streams for sure if its just this same change

Reply:
Completed, was easy to support.